components/dropbox/actions/download-file-to-tmp/download-file-to-tmp.mjs

```diff
@@ -0,0 +1,47 @@
+import dropbox from "../../dropbox.app.mjs";
+import common from "../common/common.mjs";
+import fs from "fs"
+import { file } from 'tmp-promise'
+
+export default {
+  ...common,
+  name: "Download File To TMP",
+  description: "Download a specific file to tmp dir. [See docs here](https://dropbox.github.io/dropbox-sdk-js/Dropbox.html#filesDownload__anchor)",
```

Comment:
```suggestion
  description: "Download a specific file to the temporary directory. [See docs here](https://dropbox.github.io/dropbox-sdk-js/Dropbox.html#filesDownload__anchor)"
```

Reply:
Changed abbreviation 'tmp' to 'temporary' for clarity (Google's Developer docs style guide)